asapdiscovery-docking/asapdiscovery/docking/workflows/workflows.py

```diff
@@ -28,6 +28,10 @@ class DockingWorkflowInputsBase(BaseModel):
         description="Path to a directory containing structures to dock instead of a full fragalysis database.",
     )
 
+    postera_molset_name: Optional[str] = Field(
+        None, description="The name of the molecule set to upload to."
+    )
+
```

Comment:
I think we have moved this to `PosteraDockingWorkflowInputs`

Reply:
So this should be deleted here

Reply:
good pickup tyvm.